### DISCUSSION TITLE
N-SPC accuracy

### DISCUSSION THREAD

**Discussion by livvy94** [vgmtrans/vgmtrans#258] [Music formats]
I made an SPC file for testing various effects and note properties. Feel free to use this to fix bugs!
This uses the version of the N-SPC engine found in EarthBound, as well as (I think?) other games.
[SPC download](https://www.dropbox.com/s/477nehki1k6pr76/N-SPC%20Test.spc?dl=0)
[Playback in snes9x](https://www.dropbox.com/s/sh4b7g77e261xpd/N-SPC%20Test.mp3?dl=0)

**EDIT: Added frame 7**

- Frame 0:
    - Testing each instrument currently loaded into memory
- Frame 1:
    - The vibrato command [E3 start speed range]
    - The note slide commands
        Fromnote [F1 start length range]
        To note [F2 start length range]
- Frame 2:
    - The channel transpose command [EA semitones]
    - The panning command [E1 panning]
    - The panning slide command [E2 length panning]
- Frame 3: 
    - A combination of the note slide commands from before, and the note-based pitch bend command [F9 start length (insert note here)]
- Frame 4:
    - The volume slide command [EE length volume]
- Frame 7: 
    - The Set Initial Drum command, loops, and channel transpose
- Frame 5:
    - Note length settings
- Frame 6:
    - Combination of some of these

**Comment by sykhro** [comment]
Thanks for the contribution, I have renamed the issue to be more appropriate for the tracker.
